nipype/interfaces/bids_utils.py

```diff
@@ -56,36 +48,33 @@ class BIDSDataGrabber(BaseInterface):
     Examples
     --------
 
-    >>> from nipype.interfaces.bids_utils import BIDSDataGrabber
-    >>> from os.path import basename
-
     By default, the BIDSDataGrabber fetches anatomical and functional images
     from a project, and makes BIDS entities (e.g. subject) available for
     filtering outputs.
 
-    >>> bg = BIDSDataGrabber()
-    >>> bg.inputs.base_dir = 'ds005/'
-    >>> bg.inputs.subject = '01'
-    >>> results = bg.run()
-    >>> basename(results.outputs.anat[0]) # doctest: +ALLOW_UNICODE
-    'sub-01_T1w.nii.gz'
+    >>> bg = BIDSDataGrabber() # doctest: +SKIP
+    >>> bg.inputs.base_dir = 'ds005/' # doctest: +SKIP
```

Comment:
you could add a dummy directory with a fake file or reuse an existing directory from the `testing/data` folder

Reply:
we want to add more testing data to nipype? we'll need a dedicated bids layout for grabbids